backend/src/api/Services/Users/UserService.cs

```diff
@@ -1,104 +1,33 @@
 ﻿using api.Data;
-using Microsoft.AspNetCore.Identity;
-using System;
-using System.Threading.Tasks;
-using System.Linq;
-using System.Collections.Generic;
 using api.Data.DTO;
 using api.Data.DAO;
-using api.Services.Tokens;
-using System.Security.Claims;
+using api.Constants;
 
 namespace api.Services.Users;
 
 public class UserService : IUserService
 {
-    private readonly UserManager<UserModel> _userManager;
-    private readonly RoleManager<IdentityRole> _roleManager;
     private readonly UsersDbContext _usersDbContext;
-    private readonly ITokenService _tokenService;
 
-    public UserService(UserManager<UserModel> userManager, RoleManager<IdentityRole> roleManager, UsersDbContext usersDbContext, ITokenService tokenService)
+    public UserService(UsersDbContext usersDbContext)
     {
-        _userManager = userManager;
-        _roleManager = roleManager;
         _usersDbContext = usersDbContext;
-        _tokenService = tokenService;
     }
 
-    public async Task<(bool IsSuccess, IEnumerable<string> Error)> Register(UserDTO user)
+    public async Task<(bool isSuccess, string? error)> CreateKeycloakUser(UserDto user)
```

Comment:
I know that I have introduced the pattern of using tuples in this way, but now that I'm a bit more expirienced I would sugest using this pattern from .NET instead:
`public async Task<bool> TryCreateKeycloakUser(UserDto user, out string? error)` 
as it is much more common

Reply:
![image](https://github.com/ksummarized/ksummarized.com/assets/45317651/561020a5-4a93-47ff-a962-f2790e3dd14e)
We are using async Tasks and in that case it is impossible to use `out` parameter.